lib/transforms/json-to-html-data.js

```diff
@@ -325,13 +325,10 @@ function getObjectType (data, links) {
   var value;
   if (data.type !== 'people' && data.attributes.name) {
     value = data.attributes.name.map((el, i, arr) => {
-      var val = el.value;
```

Comment:
For all the values? I add to keep a similar code for separate the occupations for a person